tofhir-log-server/src/main/scala/io/tofhir/log/server/endpoint/ExecutionEndpoint.scala

```diff
@@ -1,41 +1,46 @@
-package io.tofhir.server.endpoint
+package io.tofhir.log.server.endpoint
 
 import akka.http.scaladsl.model.headers.RawHeader
 import akka.http.scaladsl.server.Directives._
 import akka.http.scaladsl.server.Route
+import io.tofhir.log.server.model.Json4sSupport._
 import com.typesafe.scalalogging.LazyLogging
-import io.tofhir.server.endpoint.JobEndpoint.{SEGMENT_EXECUTIONS, SEGMENT_JOB, SEGMENT_LOGS}
-import io.tofhir.server.interceptor.ICORSHandler
-import io.tofhir.server.model.Json4sSupport._
-import io.tofhir.engine.util.FhirMappingJobFormatter.formats
-import io.tofhir.server.model.ToFhirRestCall
-import io.tofhir.server.service.ExecutionService
+import io.tofhir.log.server.config.WebServerConfig
+import io.tofhir.log.server.interceptor.{ICORSHandler, IErrorHandler}
+import io.tofhir.log.server.service.ExecutionService
+import ExecutionEndpoint.{SEGMENT_EXECUTIONS, SEGMENT_JOB, SEGMENT_LOGS, SEGMENT_PROJECTS}
 
-
-class JobEndpoint() extends LazyLogging {
+class ExecutionEndpoint(webServerConfig: WebServerConfig) extends ICORSHandler with IErrorHandler with LazyLogging {
 
   val executionService: ExecutionService = new ExecutionService()
 
-  def route(request: ToFhirRestCall): Route = {
-    pathPrefix(SEGMENT_JOB) {
-      val projectId: String = request.projectId.get
-      pathPrefix(Segment) { jobId: String =>
-        pathPrefix(SEGMENT_EXECUTIONS) { // Operations on all executions, jobs/<jobId>/executions
```

Comment:
There are 2 comments like this -> // Operations on all executions, jobs/<jobId>/executions

Are there any operations on these endpoint? If not, let's update the comments.